solc/CommandLineInterface.cpp

```diff
@@ -301,7 +271,20 @@ void CommandLineInterface::handleGasEstimation(string const& _contract)
 	}
 }
 
-bool CommandLineInterface::parseArguments(int argc, char** argv)
+void CommandLineInterface::createFile(string const& _fileName, string const& _data)
```

Comment:
Slight code duplication with CommonIO.cpp. Perhaps we could add an option to writeFile in CommonIO as to whether the data should be protected or not?